src/main/java/com/mitchej123/hodgepodge/mixins/fixIc2Hazmat/MixinIc2Hazmat.java

```diff
@@ -0,0 +1,18 @@
+package com.mitchej123.hodgepodge.mixins.fixIc2Hazmat;
+
+import ic2.core.item.armor.ItemArmorHazmat;
+import net.minecraft.entity.EntityLivingBase;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import gregtech.api.util.GT_Utility;
+
+@Mixin(value = ItemArmorHazmat.class, remap = false)
+public class MixinIc2Hazmat {
+    @Overwrite()
```

Comment:
Why do you use Overwrite instead on Injecting in HEAD? You could break the vanilla logic

Reply:
never did mixins stuff so I asked mitch for help, that's what he recommended
I don't really know how to work with mixins, I tried, if it's somehow wrong I can try redoing it but I need some help

Reply:
Because you are completely taking over the function and replacing every and single instruction. What else are you going to use?

Reply:
It's a simple function that we want to completely replace the logic for, overwrite makes sense in this context.